### PR TITLE
osu! Coffee Hour: osu!lazer newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
☕ *sip* ☕

### CODE REVIEW

File: news/2017-10-11-osu-coffee-hour-osulazer.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "osu! Coffee Hour: osu!lazer"
+date: 2017-10-11 06:00:00 +0000
+---
+
+The osu! Coffee Hour is back again for another sitting, this time showcasing one of the most popular osu!-related things on everyone's mind - **osu!lazer**. Have your questions about the future of the game's.. everything answered in this week's session, featuring **peppy** and **smoogipoo**!
```
Comment (Raidencio) [inline news/2017-10-11-osu-coffee-hour-osulazer.md]: missing a period for the ellipsis

Comment (TPGPL) [reply news/2017-10-11-osu-coffee-hour-osulazer.md]: osu!-related topics?

Comment (TPGPL) [reply news/2017-10-11-osu-coffee-hour-osulazer.md]: the future of the game

File: news/2017-10-11-osu-coffee-hour-osulazer.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+[![](/wiki/shared/news/banners/coffee-hour.jpg)](http://itsalmo.st/#osulazershowcasetime)
+
+
+## This week's theme
+
+This week we'll be bringing you a showcase featuring what's new in osu!lazer, the brand new client-slash-platform that is quite literally, the future of the game.
```
Comment (TPGPL) [inline news/2017-10-11-osu-coffee-hour-osulazer.md]: This header is copying the main one I think

File: news/2017-10-11-osu-coffee-hour-osulazer.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Joining me [(HappyStick)](https://osu.ppy.sh/u/happystick) in making this happen is none other than the lead developer himself, [peppy](https://osu.ppy.sh/u/2), and one of the most active osu!devs involved with the project, [smoogipoo](https://osu.ppy.sh/u/1040328).
```
Comment (TPGPL) [inline news/2017-10-11-osu-coffee-hour-osulazer.md]: Joining me?
If so, add "Joining me, Happystick,

File: news/2017-10-11-osu-coffee-hour-osulazer.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+[![](/wiki/shared/news/banners/coffee-hour.jpg)](http://itsalmo.st/#osulazershowcasetime)
+
+
+## This week's theme
+
+This week we'll be bringing you a showcase featuring what's new in osu!lazer, the brand new client-slash-platform that is quite literally, the future of the game.
+
+Joining me [(HappyStick)](https://osu.ppy.sh/u/happystick) in making this happen is none other than the lead developer himself, [peppy](https://osu.ppy.sh/u/2), and one of the most active osu!devs involved with the project, [smoogipoo](https://osu.ppy.sh/u/1040328). 
+
+Our goal will be to teach you at least a couple new things about the project, even if you've already checked it out in the past.
```
Comment (TPGPL) [inline news/2017-10-11-osu-coffee-hour-osulazer.md]: Our goal is

File: news/2017-10-11-osu-coffee-hour-osulazer.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## Questions?
+
+As has been the theme, **we're looking for your questions!** 
+
+Anything related to osu!lazer goes, so post your questions in the comments below, and we'll choose the top upvoted comments to present to the guests for discussion on the show.
+
+## When is this happening?
+
+The next episode of the osu! Coffee Hour will be happening this Friday, the [13th of October, at 11PM UTC](http://itsalmo.st/#osulazershowcasetime) over at [osulive](http://www.twitch.tv/osulive).
```
Comment (TPGPL) [inline news/2017-10-11-osu-coffee-hour-osulazer.md]: is happening

**Comment by Ephemeralis** [comment]
Applied most of these.

**Pull request merged**